PR for Booking Service Proxy v.1

hannahmanfredi opened the pull request:
Hi! Here are the instructions for running my Proxy:
1. clone down my repo from 'dev' branch
2. npm install
3. run my server for the Booking Service on localhost PORT: 3002
3. to test my proxy run:
//curl http://localhost:3002/booking?campId=0

Commit:
Delete index.js, cut new dev -b

Commit:
Write Proxy code in index.js at PORT: 3006 and test via CL curl commands

Commit:
Add proxy.html and change PORT to 2000

Commit:
Change Gal to Booking

Commit:
Test with curl http://localhost:3002/booking?campId=0 while booking
server running and return data!!!!

Commit:
Update my proxy.html file

Commit:
Stop using index.js file in order to reach my BMR

Commit:
Bring back my index.js file

Commit:
Link index.js to my proxy.html

Commit:
Update reviews app id to reviews

kimberlyjk commented on proxy.html:
```diff
@@ -0,0 +1,19 @@
+    <div id="reviews"></div>
+    <div id="app"></div>
+  </body>
+  <script type="text/javascript" src="index.js"></script>
```
is this the code for your proxy?

kimberlyjk commented on proxy.html:
```diff
@@ -0,0 +1,19 @@
+    <div id="booking"></div>
+    <div id="overview"></div>
+    <div id="reviews"></div>
+    <div id="app"></div>
```
this looks awesome

kimberlyjk commented on index.js:
```diff
@@ -40,6 +39,7 @@ app.listen(PORT, HOST, () => {
-//curl localhost:3006/localhost:3002/booking/1
\ No newline at end of file
+//test my proxy:
+//curl http://localhost:3002/booking?campId=0
```
this is not going to test your proxy becuase its making a direct request to the service, whic. is at 3002. a proxy test should lead with http://localhost:2000

kimberlyjk commented on index.js:
```diff
@@ -1,20 +1,19 @@
+const PORT = 2000;
 const HOST = "localhost";
-const API_SERVICE_URL = "https://localhost:3002/booking";
+const BOOKING_SERVICE_URL = "https://localhost:3002";
```
smart!

kimberlyjk reviewed (changes requested):
Nice work on the HTML page. in regards to your server that you've set up, I don't think you're there quite yet. I am not too familiar with http-proxy-middleware, though what you've written looks and makes sense... The test you have in your instructions isn't corect though, and I've made that comment in the code as well as told you verbally. :D

Commit:
Add Public directory and serve from index.js (express server). Still
getting CORS issue which I can only resolve by each service fixing it
own CORS errors.

hannahmanfredi committed:
Merge branch 'main' into dev

Commit:
Comment out scripts for reviews and photogallery for testing with
overviews/booking only
